src/VirtoCommerce.Platform.Web/Controllers/Api/SecurityController.cs

```diff
@@ -662,6 +662,30 @@ public async Task<ActionResult<UserApiKey[]>> DeleteUserApiKeys([FromQuery] stri
             return Ok();
         }
 
+        /// <summary>
+        /// Verify user email
+        /// </summary>
+        /// <param name="userName"></param>
+        [HttpPost]
+        [Route("users/{userName}/verifyEmail")]
+        [Authorize(PlatformConstants.Security.Permissions.SecurityVerifyEmail)]
+        public async Task<ActionResult> VerifyEmail([FromRoute] string userName)
```

Comment:
better to change to SendVerificationEmail or TriggerVerification or someting like that
VerifyEmail is misleading

Reply:
renamed